local_subscriber.py

```diff
@@ -27,14 +29,14 @@ def configure(self, context):
         self._subscriber = NioSubscriber(
             self._subscriber_handler,
             topic="{}.{}".format(self.local_identifier(), self.topic()))
-
-    def start(self):
-        """ Start the block by opening the underlying subscriber """
-        super().start()
-        self._subscriber.open()
+        self._subscriber.open(on_connected=self.conn_on_connected,
```

Comment:
This will cause errors if using another communication module (or an older version of the pk communication module). Instead, can we put this in a try/except and catch whatever exception gets raised on an invalid argument? If we can't use the callbacks we should assume we're connected and potentially log an `INFO` or `WARNING` about the fact that we can't detect the state of the comms because of an outdated module. I would probably prefer INFO for that, not sure it needs to be a WARNING but I could be convinced otherwise.

Reply:
going for WARNING here, it is what we normally do when running outdated versions